cot/src/session/store/redis.rs

```diff
@@ -122,6 +119,11 @@ impl RedisStore {
     /// The returned `Connection` implements
     /// `AsyncCommands` so you can run Redis commands directly.
     ///
+    ///  # Errors
+    ///
+    ///  Returns [`RedisStoreError::PoolConnection`] if it fails to get a valid
+    /// redis connection from the connection pool.
```

Comment:
```suggestion
    /// # Errors
    ///
    /// Returns [`RedisStoreError::PoolConnection`] if it fails to get a valid
    /// redis connection from the connection pool.
```

nitpick: unnecessary spaces